task9.cpp

```diff
@@ -0,0 +1,27 @@
+#include <iostream>
+
+using namespace std;
+
+int calculate(int a, int b, char action) {
+    switch(action) {
+        case '+': return a + b;
+        case '-': return a - b;
+        case '*': return a * b;
+        case '/': {
```

Comment:
Advice, you don't need the curly brackets here even though they do make it more clear to read